groups/bdl/bdlb/bdlb_random.t.cpp

```diff
@@ -217,7 +220,8 @@ int main(int argc, char *argv[])
       case 3: {
         // --------------------------------------------------------------------
         // USAGE EXAMPLE
-        //   Extracted from component header file.
+        //   Extracted from component header file. For 'generatePcg' (PCG)
+        //   usage example please refer bdlb_pcgrandomgenerator.t.cpp.
```

Comment:
```suggestion
        //   Extracted from component header file. 
```
This is not really relevant for the test driver.  The PCG stuff should be just be linked in component documentation in `@SEE_ALSO` section - search the code for where it goes and how it's formatted.